tests/installation/scc_registration.pm

```diff
@@ -87,6 +87,7 @@ sub run {
 sub post_fail_hook {
     my ($self) = @_;
     $self->SUPER::post_fail_hook;
+    y2_installbase::save_upload_y2logs;
```

Comment:
why not move logic this to lib/y2_installbase.pm ?

Reply:
y2_installbase.pm has the post_fail_hook with the save_upload_y2logs. While it won't do the upload for the SUPER which will make the own post_fail_hook to be a independent func but not inherit from parent.

Reply:
Hmm, I don't really understand why we need it, saving yast logs is a must for any of the installation modules. With this line added we will upload it twice.

Reply:
Ok, I think it is for my misunderstanding for the SUPPER. I will remove the change in tests/installation/scc_registration.pm